components/feature/media/src/main/res/drawable/mozac_feature_media_playing.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
nit: license header

Reply:
Those are not our own icons yet - they are from the material design icon set.